iceoryx_examples/iceperf/README.md

```diff
@@ -16,27 +16,29 @@ At the end of the benchmark, the average latency for each payload size is printe
 ## Run iceperf
 
 Create three terminals and run one command in each of them.
-The order is first the RouDi daemon, then iceperf-laurel which is the leader in this setup
-and then iceperf-hardy for doing the ping pong measurements with iceperf-laurel. 
-You can set the number of measurement iterations (number of roundtrips) with a command line paramter
-of iceperf-laurel (e.g. `./iceperf-laurel 100000`)
+The order is first the RouDi daemon, then `iceperf-bench` as follower and leader with the `-f` or
```

Comment:
It reads like the follower/leader should start at the same time but wasn't the order like:
1. roudi
2. leader
3. follower

If so could you make this more clear. Maybe with an enumaration like above?

Reply:
the order can now be arbitrary. Just need to update the README